benchmark/dgram/bind-params.js

```diff
@@ -19,19 +19,27 @@ function main(conf) {
   if (port !== undefined && address !== undefined) {
     bench.start();
     for (let i = 0; i < n; i++) {
-      dgram.createSocket('udp4').bind(port, address).unref();
+      dgram.createSocket('udp4').bind(port, address)
+        .on('error', () => {})
```

Comment:
Perhaps pull this out to avoid creating `n` functions? I'm not sure what V8 optimizes away these days.